samples/Azure.Analytics.Purview.Account/Customizations.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+using Azure.Core;
+
+namespace Azure.Analytics.Purview.Account
+{
+    [CodeGenClient(ParentClient = typeof(AccountsClient))]
+    public partial class PurviewAccountCollections { }
```

Comment:
This has become a bit unintuitive: user has to be aware of the different naming rules for subclients to apply customization. In this example, the top level name would be `CollectionsClient`, but customization type should be `PurviewAccountCollections`. As an alternative, we can always specify old client name, e.g.:
```cs
    [CodeGenClient("CollectionsClient", ParentClient = typeof(AccountsClient))]
    public partial class PurviewAccountCollections { }
```

That would also solve the naming problem cause user will always specify the name of subclient explicitly.

Reply:
Swagger doesn't give us any information except for the case when we have "factory" top level client that has only method to create subclients based on their operation group names